app/services/mailer.js

```diff
@@ -0,0 +1,36 @@
+const mailer = require('nodemailer');
+const { EMAIL_SENDER, EMAIL_GENERAL_ERROR } = require('../../config/constants');
+const logger = require('../logger');
+
+const welcomeMessage = (name, lastName, email) => ({
+  from: `<${EMAIL_SENDER}>`,
+  to: email,
+  subject: 'Welcome ✔',
+  text: `Welcome ${name} ${lastName}`,
+  html: `<h1>Witter API</h1>
```

Comment:
If your HTML is too large, is necessary to create another file, then import it.

Reply:
Great approach.